k8s/helmfile/helmfile.yaml

```diff
@@ -183,7 +183,7 @@ releases:
   - name: queryservice-gateway
     namespace: default
     chart: wbstack/queryservice-gateway
-    version: 0.1.3
+    version: 0.2.0
```

Comment:
I would think it's ok to set this for production as well without updating the image just yet as all it does is set an extra env var that is not picked up by `2.2`

Reply:
yeah fine; I think you can self merge this if you wanted